blockchain/handlers.go

```diff
@@ -198,7 +198,7 @@ func DeployBlocksLookUpAndUpdate(blockchain string) error {
 				sem <- struct{}{}
 				defer func() { <-sem }()
 
-				client, err := NewClient(chain, BlockchainURLs[chain], 4)
+				client, err := NewClient(chain, rpcUrl, rpcTimeout)
```

Comment:
What if blockchain provided for ethereum and rpcUrl for polygon, will we write polygon data in ethereum tables?
I think `BlockchainURLs` built to prevent this.

Reply:
At least I would modify `BlockchainURLs` to some switch-case with rpc call to get chain id and if it is correct ethereum - then start or throw exception.

Reply:
Or even we can reduce flag `chain` to fetch chain id from rpc

Reply:
chain responsible for selection of db write operation. RPC for connecting to blockchain. Only 1 possible solution is have mapping with chain ids or in code or in database.